src/main/java/uk/gov/hmcts/divorce/document/model/DocumentType.java

```diff
@@ -209,6 +209,30 @@ public enum DocumentType implements HasLabel {
     @JsonProperty("coGrantedCoversheetApp2")
     CONDITIONAL_ORDER_GRANTED_COVERSHEET_APP_2("Respondent/Applicant 2 Conditional order granted cover letter"),
 
+    @JsonProperty("judicialSeparationGrantedCoversheet")
```

Comment:
As mentioned in other part of this review, we don't need to overwrite the type (as NoC for examples)